db/migrate/20160513183624_create_evaluations.rb

```diff
@@ -0,0 +1,11 @@
+class CreateEvaluations < ActiveRecord::Migration
+  def change
+    create_table :evaluations do |t|
+      t.float :answer
```

Comment:
I wasn't sure if you could average integers to a decimal point, so went with floats to be safe